webpack-main-prod.config.js

```diff
@@ -0,0 +1,10 @@
+const merge = require('webpack-merge');
+const common = require('./webpack-main.config.js');
+
+module.exports = merge(common, {
+    mode: 'production',
+    devtool: 'source-map',
```

Comment:
If there is javascript bug on live, having a source-map would help to identify the error much quicker? Initially my concern was that this exposes the source code, but TEAMMATES is an open sourced project.

In terms of performance, I think full `source-map` is okay for production build because we only run it once.